examples/kuesa/tonemapping/qml/SceneEntity.qml

```diff
@@ -0,0 +1,182 @@
+            specular: TextureLoader {
+                source: _assetsPrefix + "pink_sunrise" + envMapFormat + "_specular" + ((!root3D.es2) ? ".dds" : "_es2.dds")
+
+                minificationFilter: Texture.LinearMipMapLinear
+                magnificationFilter: Texture.Linear
+                wrapMode {
+                    x: WrapMode.ClampToEdge
+                    y: WrapMode.ClampToEdge
+                }
+                generateMipMaps: false
+            }
+        }
+    ]
+
+    Entity {
+        Transform {
+            id: transform
+            property real angle: 0.0
+
+            readonly property real radianAngle: angle * 2 * Math.PI / 360
+            readonly property real radius: 10
+            readonly property real x: radius * Math.cos(radianAngle)
+            readonly property real y: 0
+            readonly property real z: radius * Math.sin(radianAngle)
+
+            scale: 10
+            translation: Qt.vector3d(x, y, z)
+
+            NumberAnimation on angle {
+                loops: Animation.Infinite
+                duration: 10000
+                from: 0
+                to: 360
+                paused: root3D.lightRotating
+            }
+        }
+
+        PointLight {
+            id: pointLight
+            color: "white"
+            intensity: 0.2
+        }
+        components: [ pointLight, transform ]
+    }
+
+    Camera {
+        id: fallbackCamera
+        position: Qt.vector3d(0.0, 0.0, 7.0)
+        upVector: Qt.vector3d(0.0, 1.0, 0.0)
+        exposure: root3D.exposure
+        viewCenter: Qt.vector3d(0.0, 0.0, 0.0)
+        aspectRatio: root3D.__winSize.width / root3D.__winSize.height
+    }
+
+    KuesaDemo.OrbitCameraController {
+        id: controller
+        camera: frameGraph.camera
+        windowSize: root3D.__winSize
+    }
+
+    Kuesa.AnimationPlayer {
+        id: helmetSpin
+        sceneEntity: root3D
+        clip: "damagedHelmet_Action"
+        loops: Kuesa.AnimationPlayer.Infinite
+        running: root3D.rotating
+    }
+
+    Kuesa.Asset {
+        id: cameraAsset
+        name: root3D.cameras.names.length ? root3D.cameras.names[0] : ""
+        collection: root3D.cameras
+        onNodeChanged: {
+            if (node)
+                node.aspectRatio = root3D.__winSize.width / root3D.__winSize.height
+        }
+    }
+
+    onFilmicEffectChanged: metallic = filmicEffect ? Kuesa.MetallicRoughnessEffect.Filmic : Kuesa.MetallicRoughnessEffect.Reinhard
+
+    Kuesa.Asset {
+        id: materialsAsset
+        name: "Material_MR"
+        collection: root3D.materials
+        onNodeChanged: node.toneMappingAlgorithm = Kuesa.MetallicRoughnessEffect.Reinhard
+    }
+
+    Binding {
+        target: materialsAsset.node
+        property: "toneMappingAlgorithm"
+        value: root3D.metallic
+    }
+
+    Kuesa.GLTF2Importer {
+        id: gltf2importer
+        sceneEntity: root3D
+        assignNames: true
+        source: _assetsPrefix + "RotatingHelmet.gltf"
```

Comment:
the scene in this example uses normal mapping but the scene doesnt have tangents. We need to generate them. For that, we need to 
`options.generateTangents: true`
otherwise looks black

Reply:
done

Reply:
on linux I couldnt see the helmet (was black) because it was missing the tangents. could you see it?

Reply:
yes it was fine. Gets dark when you turns on cinematic effect but that's expected I think. Not no difference between with or without tangents

Reply:
odd. was failing in linux